connectors/bitcoin.go

```diff
@@ -817,3 +818,18 @@ func getOpCodeFromInt(val int) byte {
 		return txscript.OP_16
 	}
 }
+
+func isP2PKH(address string) bool {
```

Comment:
those are fine, but might be worth to create unit tests for these functions, in a different PR though.